pkg/csi/cinder/driver.go

```diff
@@ -54,15 +55,18 @@ type CinderDriver struct {
 }
 
 func NewDriver(nodeID, endpoint, cluster string) *CinderDriver {
-	klog.Infof("Driver: %v version: %v", driverName, version)
 
 	d := &CinderDriver{}
 	d.name = driverName
 	d.nodeID = nodeID
-	d.version = version
+	// cinder CSI don't have a driver version now, so use spec CSI version instead
```

Comment:
@jichenjc the version field specifies the driver version, not the spec version. we just maintain it on par doesnt necessarily mean its spec versio